extensions/tower/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "tower",
+  "title": "Tower",
```

Comment:
```suggestion
  "title": "Tower Repositories",
```

Reply:
Considering that at the moment you only support one command, we encourage to have more specific extension name. Once you add more, you can move back to just "Tower"